openmdao/core/tests/test_problem.py

```diff
@@ -1140,6 +1139,8 @@ def setup(self):
         self.assertTrue(isinstance(top.model.sub.nonlinear_solver, NewtonSolver))
         self.assertTrue(isinstance(top.model.sub.linear_solver, ScipyKrylov))
 
+
```

Comment:
You've added a few extra carriage returns here.

Reply:
fixed. Not sure how that happened